packages/terra-disclosure-manager/docs/README.md

```diff
@@ -0,0 +1,209 @@
+# Terra Disclosure Manager
+
+The DisclosureManager is a stateful component used to manage disclosure presentation.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-disclosure-manager`
+
+## Usage
+
```

Comment:
Don't forget this section 😄

Reply:
I added subsections to divide up the Usage section; I think this reads a little more clearly when rendered.